boot.c

```diff
@@ -200,6 +208,12 @@ eyrie_boot(uintptr_t dummy, // $a0 contains the return value from the SBI
 
   /* initialize free memory */
   init_freemem();
+
+  /* initialize user stack */
+  init_user_stack();
+
+  /* settup user stack env/aux */
+  setup_start((void*)runtime_va_start);
```

Comment:
@dkohlbre I think this is wrong isn't it? 
the stack pointer is supposed to be updated?